bauh/view/qt/window.py

```diff
@@ -777,7 +778,7 @@ def _update_type_filters(self, available_types: dict = None, keep_selected: bool
                     icon = self.cache_type_filter_icons.get(app_type)
 
                     if not icon:
-                        icon = load_icon(icon_path, 14)
+                        icon = load_icon(icon_path, 18)
```

Comment:
Hi @octopusSD. I've been testing the application with your changes, and lines 161 and 781 make the icons bigger for non-hdpi resolutions. Do these new sizes make a big difference for HDPI resolutions ? ( I'm currently with no HDPI device to test, that's why I'm asking ). The pictures below show how non-hdpi devices are affected:

**Without changes**
![no_pr](https://user-images.githubusercontent.com/4312414/72009419-a2d86400-3234-11ea-9beb-de7496459578.png)

**With changes**
![pr_1](https://user-images.githubusercontent.com/4312414/72009478-c3082300-3234-11ea-893b-0e8c67e98661.png)